WORKSPACE

```diff
@@ -68,10 +68,27 @@ _go_image_repos()
 
 ## Pull some standard base images
 container_pull(
-    name = "alpine",
-    registry = "gcr.io",
-    repository = "jetstack-build-infra/alpine",
-    tag = "3.7-v20180822-0201cfb11",
+    name = "alpine_linux-amd64",
+    digest = "sha256:cf2412cab4f40318e722d2604fa6c79b3d28a7cb37988d95ab2453577417359a",
+    registry = "index.docker.io",
+    repository = "munnerz/alpine",
```

Comment:
Should this point to `munnerz/alpine` or upstream Alpine?

Reply:
`munnerz/alpine`: https://github.com/munnerz/alpine

Which is basically just the upstream Alpine but with ca-certificates installed, and built for amd64/arm64/arm 😄 

Thanks for the review!